src/osd/OSD.cc

```diff
@@ -3948,6 +3948,7 @@ void OSD::load_pgs()
   }
 
   int num = 0;
+  //this place can be improve from single thread to multithread
```

Comment:
So it is really a proposal instead of a fix?

Reply:
> So it is really a proposal instead of a fix?

after optimization, the load_pg execution time can be controlled within 1 minute.